contrib/docker/Dockerfile

```diff
@@ -0,0 +1,78 @@
+FROM alpine:3.9
+
+MAINTAINER Andrew V. Jones "andrew.jones@vector.com"
+
+ENV LC_ALL en_US.UTF-8
+ENV LANG en_US.UTF-8
+
+#
+# Deps needed only for build
+#
+ENV BUILD_DEPS \
+    alpine-sdk \
+    cmake \
+    cython \
+    cython-dev \
+    g++ \
+    gcc \
+    git \
+    make \
+    python-dev \
+    sed
+
+#
+# Deps needed for running afterwards
+#
+ENV PERSISTENT_DEPS \
+    bash \
+    libgcc \
+    libstdc++ \
+    python
+
+#
+# Steps to build-up our image
+#
+RUN apk upgrade --update && \
+    apk add --no-cache --virtual .build-deps $BUILD_DEPS && \
+    apk add --no-cache --virtual .persistent-deps $PERSISTENT_DEPS && \
+    # \
+    # Musl does not provide /usr/include/sys/unistd.h, which Lingeling needs \
+    # \
+    echo "#include <unistd.h>" > /usr/include/sys/unistd.h && \
+    git clone https://github.com/Boolector/boolector.git && \
+    cd boolector && \
+    # \
+    # Musl does not provide quite enough of glibc, so we pretend we're Windows \
```

Comment:
As already commented, it would be great if we don't need the Windows workaround for this build.

Reply:
I can look at this, but I've only ever worked with Alpine-based images, so I'll need to understand how to work with (e.g.,) Debian ones.

Reply:
I would also prefer an Alpine-basd image due to its size. However, I'm not sure if we can fix Boolector and its dependencies to be compatible with musl.